### ISSUE TITLE
[Bug] Custom Webpack Plugins are run BEFORE Electron build is complete

### ISSUE THREAD

**Issue by micheleangioni** [quasarframework/quasar-cli#93]
### Software version

OS: Debian 9
Node: v8.9.4
NPM: 5.7.1
Quasar CLI: v0.15.9

### What did you get as the error?

When Webpack runs the Custom Plugins added in 

```
electron: {
  extendWebpack (cfg) {
    // <<<--- CUSTOM PLUGINS ADDED HERE --- >>>
  }
}
```
are run before the Electron build is complete. The `\electron-mat` folder contains only the `Unpackaged` folder and not yet, for example, the bundled linux folder.

This is causing us troubles into upgrading to Quasar v0.15, because in v0.14 we could customize our builds my editing the scripts files of the `/build` folder.

### What were you expecting?

I would expect the custom Webpack Plugins to be run **after** the complete build process, otherwise it is not possible to perform operations on the builds, such as zip them.

### What steps did you take, to get the error?

Just add a custom plugin, for example a Webpack Zip Plugin, and try to zip the Electron build. The problem is that it doesn't exist.

### Proposed solutions

A simple solution would be to allow some custom code to run after the Electron build is complete. A new Issue will soon be opened to propose this new feature.

**Comment by rstoenescu** [comment]
Pushed commit which adds quasar.conf > electron.beforePackaging()
** Available in next release: 0.15.13**

Examples:

```js
electron: {
  beforePackaging ({ appPaths, unpackagedDir }) {
    // do something...
  }
}
```
```js
electron: {
  beforePackaging ({ appPaths, unpackagedDir }) {
    return new Promise((resolve, reject) => {
      .... resolve() ...
    })
  })
}
```

Please note that the Electron build is a three step process. First step is to bundle the renderer process. Second is to bundle the main process. Finally, last step is to use Electron Packager to package the app into an executable.
`beforePackaging` is executed between step 2 and step 3.

Anything after step 3 can easily be executed after Quasar CLI finishes its job.
